config.d

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env dub
```

Comment:
Make this executable, in case someone wants to run it as a script on POSIX systems.